scripts/west_commands/patch.py

```diff
@@ -93,15 +99,17 @@ def do_add_parser(self, parser_adder):
         parser.add_argument(
             "-b",
             "--patch-base",
-            help="Directory containing patch files",
+            help=f"""
+                Directory containing patch files (relative to module dir,
+                default: {WEST_PATCH_BASE})""",
             metavar="DIR",
             default=_WEST_MANIFEST_DIR / WEST_PATCH_BASE,
             type=Path,
         )
         parser.add_argument(
             "-l",
             "--patch-yml",
-            help="Path to patches.yml file",
+            help=f"Path to patches.yml file (relative to module dir, default: {WEST_PATCH_YAML})",
```

Comment:
Technically, this doesn't need to be relative, but the default probably should be